src/Goerli-DssSpell.t.sol

```diff
@@ -135,15 +136,15 @@ contract DssSpellTest is GoerliDssSpellTestBase {
         assertTrue(spell.done());
 
         // Insert new collateral tests here
-        // checkIlkIntegration(
-        //     "TOKEN-X",
-        //     GemJoinAbstract(addr.addr("MCD_JOIN_TOKEN_X")),
-        //     ClipAbstract(addr.addr("MCD_CLIP_TOKEN_X")),
-        //     addr.addr("PIP_TOKEN"),
-        //     true,
-        //     true,
-        //     false
-        // );
+        checkIlkIntegration(
+            "RETH-A",
+            GemJoinAbstract(addr.addr("MCD_JOIN_RETH_A")),
+            ClipAbstract(addr.addr("MCD_CLIP_RETH_A")),
+            addr.addr("PIP_RETH"),
+            true,
+            true,
+            false
```

Comment:
Since Solidity does not support named parameters yet, probably it's a good idea to tag multiple boolean commands like this for clarity:

```suggestion
            /* _isOSM             */ true,
            /* _checkLiquidations */ true,
            /* _transferFee       */ false
```